c/common.h

```diff
@@ -69,3 +76,90 @@ int extract_witness_lock(uint8_t *witness, uint64_t len,
   *lock_bytes_seg = MolReader_Bytes_raw_bytes(&lock_seg);
   return CKB_SUCCESS;
 }
+
+/* a and b are since value,
+ return 0 if a is equals to b,
+ return -1 if a is less than b,
+ return 1 if a is greater than b */
+int epoch_number_with_fraction_cmp(uint64_t a, uint64_t b) {
```

Comment:
Can we bump this compare function to `ckb_syscalls.h` or a different utility file? I think it could be useful in many places.

Reply:
Sure, I think to put it in another `utils.h` is good, don't mix the utilities with syscalls.